docs/index.rst

```diff
@@ -1,7 +1,19 @@
 Welcome to Craft Providers' documentation!
 ===========================================
 
-Here you will find all of the provider documentation...
+Craft Providers is a Python package for managing software builds in containers
+on behalf of tools using the Craft Parts framework.
```

Comment:
Strictly speaking, craft-providers can be used without craft-parts (lpcraft, for example).  Although craft-providers does integrate nicely with the other craft libraries.